test/studies/shootout/submitted/spectralnorm.chpl

```diff
@@ -38,7 +38,7 @@ proc multiplyAv(v: [?Dv], Av: [?DAv]) {
 }
 
 //
-// Compute A-transpose * v ('Atv').
+// Compute A-tranpose * v ('Atv').
```

Comment:
Stringent, to keep the diffs clean.  The version in the release (from which any future submissions will hopefully be derived) spells it correctly.